packages/cmf/package.json

```diff
@@ -52,7 +52,7 @@
     "invariant": "^2.2.4",
     "lodash": "^4.17.21",
     "nested-combine-reducers": "^1.2.2",
-    "path-to-regexp": "^3.3.0",
+    "path-to-regexp": "^8.2.0",
```

Comment:
be sure you have tested an existing CMF application with the router 🙏 
There are unit test that should be good but we never know.
By the way there are still a 6.0 in the lock file, so the security issue will stay. Do you know where it comes from ?

Reply:
Thanks for the comment!
I checked playground and vite-playground which works fine.
But version 6 comes from multiple packages 😨 :
```
=> Found "path-to-regexp@6.3.0"
info Has been hoisted to "path-to-regexp"
info Reasons this module exists
   - "workspace-aggregator-8113ffd5-c4b4-4412-9f5c-5bcfa979cb8e" depends on it
   - Hoisted from "_project_#@talend#ui-playground-vite#vite-plugin-mock#path-to-regexp"
   - Hoisted from "_project_#@talend#react-containers#msw#path-to-regexp"
   - Hoisted from "_project_#@talend#react-bootstrap#sinon#nise#path-to-regexp"
 
```
on top of that, `express` uses an even older version:
```
=> Found "express#path-to-regexp@0.1.10"
info This module exists because "_project_#@talend#ui-playground-vite#express" depends on it.
```
Unfortunately, express doesn’t have a newer, safe release to address this issue. There’s no way to resolve this alert at the moment.

Reply:
In this case, can we only upgrade our direct dependency on `path-to-regexp` ?